openfisca/api_config.ini

```diff
@@ -0,0 +1,52 @@
+# OpenFisca-Web-API - Development environment configuration
+#
+# The %(here)s variable will be replaced with the parent directory of this file.
+
+[DEFAULT]
+debug = true
+# Uncomment and replace with the address which should receive any error reports
+email_to = mes-aides@sgmap.fr
+smtp_server = localhost
+error_email_from = openfisca-mes-aides@sgmap.fr
+error_from = openfisca-mes-aides@sgmap.fr
+
+[server:main]
+use = egg:Paste#http
+host = 0.0.0.0
+port = %PORT%
+
+[app:main]
+use = egg:OpenFisca-Web-API
+country_package = openfisca_france
+log_level = DEBUG
+reforms =
+	openfisca_france.reforms.aides_cd93.aides_cd93
```

Comment:
Paris is not a reform anymore, it's an extension managed by adding the directory in the right place. See [here](https://github.com/sgmap/mes-aides-ui/pull/319/files#diff-60254338249f657a0a83f98258a56bfeR85).

 Ideally we would also do the same for CD93, but as it's not moving much, i don't think it's worth investing time on it.